setup.sh

```
@@ -0,0 +1,219 @@
+    exit 1
+fi
+
+
+# Function to check if llama-stack is running
+check_llama_stack_running() {
+    local port=${1:-8321}
+    # Try multiple endpoints to check if server is ready
+    local endpoints=("/v1/models" "/v1/health" "/health")
+    
+    for endpoint in "${endpoints[@]}"; do
+        local status_code=$(curl -s -o /dev/null -w "%{http_code}" "http://localhost:$port$endpoint" 2>/dev/null)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)